extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -512,13 +561,21 @@ export class AmpAutocomplete extends AMP.BaseElement {
   renderResults_(filteredData, container, input) {
     let renderPromise = Promise.resolve();
     this.resetActiveElement_();
-    if (this.templateElement_) {
-      renderPromise = this.templates_
-        .renderTemplateArray(this.templateElement_, filteredData)
-        .then(renderedChildren => {
-          renderedChildren.map(child => {
+    if (this.hasTemplate_) {
+      renderPromise = this.ssrTemplateHelper_
+        .applySsrOrCsrTemplate(this.element, filteredData)
+        .then(rendered => {
+          const elements = isArray(rendered)
+            ? rendered
+            : toArray(rendered.children);
+          elements.forEach(child => {
             if (child.hasAttribute('data-disabled')) {
               child.setAttribute('aria-disabled', 'true');
+            } else {
+              userAssert(
+                child.hasAttribute('data-value'),
```

Comment:
I think this might be a breaking change.

Reply:
Changed to `user().warn()`